ASWF/meetings/working_groups/configs/2020-11-03.md

```diff
@@ -0,0 +1,147 @@
+      be more confusing. Would not use OCES.
+    - Mark: What would become the display space? The generator draws 
+      connections between the ACES blobs and the newer transforms skip the OCES 
+      step.
+    - Doug: Display telemetry. New transforms go through that but don't break 
+      it out as external interface in CTL, for historical reasons.
+    - Mark: That will make it difficult to make generator which looks at CTL to 
+      generate. Not married to that plan but seems sensible for tracking 
+      upstream. Also happy to abandon it for config expressed in code if that's 
+      more useful.
+    - Doug: Was thinking could use generator to identify the ACES end and 
+      display code value end, but not necessarily use the OCES interface. For 
+      output transforms, some go direct from ACES and others through OCES, but 
+      we could handle them as if they went from ACES to device.
+    - Mark: At moment, generator doesn't parse CTL code, just looks at URN in 
+      header of files, so no way to track transforms or chains between functions.
+    - Doug: We know anything that expects OCES as input wants to include RRT.
+    - Mark: So use RRT as point?
+    - Doug: Part of problem is the RRT does not produce display referred 
+      imagery, which has caused confusion. Produces idealized output referred 
+      intermediate. Actual display referred colorimetry requires first part of 
+      ODT. RRT doesn't provide technical benefit. Think we can ignore RRT, and 
+      for transforms with OCES input, go from ACES to OCES.
+    - Mark: Not sure benefit to tracking CTL to get these relationships if we 
+      need to implement transforms. Maybe skip checking of CTL and 
+      programmatically build transforms.
+    - Michael: Could use generator for testing if not for config generation. To 
+      do image comparison etc.
+    - Doug: Thomas' generator found issues which have value to go through and 
+      report the problems. Agree it may not make sense to follow it strictly 
+      from built graph. Needs to be manual intervention at some point. Agree 
+      with treating it more as CI process.
+    - Mark: Yeah, to get the names from it and try to call those transforms. 
+      Will wait until display referred spaces are setup in ACES context.
+    - Doug: Yes, will help for me to take one transform and show how it works 
+      in OCIO. Will also reduce amount of code. The way I'm proposing it is 
+      removing a lot of the redundancy in the current CTL code. If you look at 
+      ODTs there are lots of commonalities, so my proposal takes that common 
+      part and factors it into the view transform. Will make the overall 
+      structure easier to follow.
+    - Michael: That would then be a point of divergence in the code. The 
+      current code for testing against upstream ACES, and a new generator for
+      a config itself.
```

Comment:
Yep, whether the Studio config generator uses the graph remains to be assessed but at best it was always intended to be a building block.